mailbox/maildir/src/main/java/org/apache/james/mailbox/maildir/MaildirFolder.java

```diff
@@ -339,12 +339,12 @@ public MaildirMessageName getMessageNameByUid(final MailboxSession session, fina
                 String uidString = String.valueOf(uid.asLong());
                 String line = reader.readLine(); // the header
                 int lineNumber = 1;
-                while ((line = reader.readLine()) != null) {
+                for (; (line = reader.readLine()) != null; lineNumber++) {
```

Comment:
I don't like that loop change. while seems more expressive that for...

Reply:
Done